XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -894,12 +894,11 @@
         {
           "UniqueId": "RadioButton",
           "Title": "RadioButton",
-          "Subtitle": "A control that allows a user to select a single option from a group of options. When radio buttons are grouped together they are mutually exclusive.",
+          "Subtitle": "A control that allows a user to select a single option from a group of options.",
           "ImagePath": "ms-appx:///Assets/RadioButton.png",
-          "Description": "Use RadioButtons to let a user choose between mutually exclusive, related options.",
+          "Description": "Use RadioButtons to let a user choose between mutually exclusive, related options. Generally contained within a RadioButtons group control.",
```

Comment:
Are RadioButton_s supposed to always used within a RadioButtons group control? Is there a benefit to using a RadioButton instead of something like a TextBlock or simply a string when I am using the RadioButtons control?

From a simplicity point of view, putting RadioButton_s inside a RadioButtons control seems a bit strange, after all those RadioButton technically have their own checked state etc, while the containing RadioButtons also has their own selectionchanged.

Reply:
Good question. The [guidance doc](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/radio-button) doesn't mention including things that are _not_ RadioButton controls. I was under the impression that RadioButton is the main thing to be used within a RadioButtons, hence these samples. 

Automagically converting text to RadioButtons is a cool function, though I can't tell if it's a feature or a bug from the guidance docs 😅 @kikisaints can you help clarify?

As it pertains to this PR - I'd like to get this pulled in before I submit a new version of the Gallery to the Store this week. Automagic conversion should probably have its own sample pending Kiki's input, so I don't think this feedback blocks this PR.

Reply:
I agree that this PR shouldn't be blocked on this. Regarding the guidance docs, I think the "automagic wrapping" seems intentional, after all there is an example that uses plain strings instead of RadioButtons and another example binding to a collection of objects which are not RadioButton controls.

Reply:
Good point. I see what you mean now. I'll update the description.